.circleci/config.yml

```diff
@@ -196,8 +267,10 @@ jobs:
 
       - restore_cache:
           keys:
-            - cache-{{ arch }}-<< parameters.pytorch_version >>-{{ .Branch }}-20210827
+            - cache-{{ arch }}-<< parameters.pytorch_version >>-{{ .Branch }}-20990005
 
+      - <<: *setupcuda113
+      - <<: *removecuda114
```

Comment:
shouldn't it remove 114 first then setup 113?

Reply:
Removing 11.4 first and installing 11.3 would result in uninstalling several shared components only to reinstall later, increasing installation time. Keeping the current order does not affect the end result and save some time